server.py

```diff
@@ -44,28 +48,28 @@ def _make_response(self, msg):
         return m
 
     @asyncio.coroutine
-    def handle_request(self, message, payload):
-        response = aiohttp.Response(
-            self.writer, 200, http_version=message.version
-        )
-        if message.path == '/dorks':
-            m = json.dumps(
-                dict(version=1, response=dict(dorks=random.sample(self.dorks, 50))),
-                sort_keys=True, indent=2
-            ).encode('utf-8')
-        elif message.path == '/event':
-            data = yield from payload.read()
-            try:
-                data = json.loads(data.decode('utf-8'))
-                path = data['path']
-                sensor_uuid = data['uuid'] if 'uuid' in data else None
-            except (TypeError, ValueError, KeyError) as e:
-                print('error parsing: {}'.format(data))
-                m = self._make_response(msg=type(e).__name__)
+    def handle_event(self, data):
+        try:
+            data = json.loads(data.decode('utf-8'))
+            path = data['path']
+            # dummy for wp-content
+            if re.match(r'/wp-content/.*', path):
+                m = self._make_response(msg=dict(detection={'name': 'wp-content', 'order': 1}))
+                return m
+            sensor_uuid = data['uuid'] if 'uuid' in data else None
+        except (TypeError, ValueError, KeyError) as e:
+            print('error parsing: {}'.format(data))
+            m = self._make_response(msg=type(e).__name__)
+        else:
+            session = yield from HttpRequestHandler.session_manager.add_or_update_session(data)
```

Comment:
I think we should store it as session. Right now you return before you store the session. You think we shouldn't store it as session?